models/spring-ai-ollama/src/main/java/org/springframework/ai/ollama/api/OllamaApi.java

```diff
@@ -122,59 +125,43 @@ public OllamaApi(String baseUrl, RestClient.Builder restClientBuilder) {
-	 * @param format (optional) The format to return the response in. Currently the only
-	 * accepted value is "json".
-	 * @param options (optional) additional model parameters listed in the documentation
-	 * for the Modelfile such as temperature.
-	 * @param system (optional) system prompt to (overrides what is defined in the Modelfile).
-	 * @param template (optional) the full prompt or prompt template (overrides what is
-	 *  defined in the Modelfile).
-	 * @param context the context parameter returned from a previous request to /generate,
-	 * this can be used to keep a short conversational memory.
-	 * @param stream (optional) if false the response will be returned as a single
-	 * response object, rather than a stream of objects.
-	 * @param raw (optional) if true no formatting will be applied to the prompt and no
-	 * context will be returned. You may choose to use the raw parameter if you are
-	 * specifying a full templated prompt in your request to the API, and are managing
-	 * history yourself.
-	 * @param images (optional) a list of base64-encoded images (for multimodal models such as llava).
-	 * @param keepAlive (optional) controls how long the model will stay loaded into memory following the request (default: 5m).
+	 * @param model The model to use for completion. It should be a name familiar to Ollama from the <a href="https://ollama.com/library">Library</a>.
+	 * @param prompt The prompt(s) to generate completions for.
+	 * @param suffix The text that comes after the inserted text.
+	 * @param system Overrides the model's default system message/prompt.
+	 * @param template Overrides the model's default prompt template.
+	 * @param context The context parameter returned from a previous call to /generate. It can be used to keep a short conversational memory.
+	 * @param stream Specifies whether the response is streaming. If false the response will be returned as a single response object, rather than a stream of objects.
+	 * @param raw If set to true, it means that no formatting will be applied to the prompt. You may choose to use the raw parameter if you are specifying a full templated prompt in your request to the API.
+	 * @param format The format to return the response in. Currently, the only accepted value is "json".
+	 * @param keepAlive Controls how long the model will stay loaded into memory following this request (default: 5m).
+	 * @param images A list of base64-encoded images accompanying this request, for multimodal models such as "llava".
+	 * @param options Model-specific options. For example, "temperature" can be set through this field, if the model supports it.
+	 *
+	 * @see <a href=
+	 * "https://github.com/ollama/ollama/blob/main/docs/api.md#generate-a-completion">Generation API</a>
+	 * @see <a href="https://github.com/ollama/ollama/blob/main/api/types.go">Ollama
+	 * Types</a>
 	 */
 	@JsonInclude(Include.NON_NULL)
 	public record GenerateRequest(
```

Comment:
It's worth considering whether we want to keep maintaining the APIs for the /generate endpoint which is not used by any Spring AI abstraction.

For now, I changed it so that it's fully in sync with the latest Ollama version, including the order of the fields to help future maintenance.